### PR TITLE
Disable SCCA task

### PR DESCRIPTION
**Pull request by yophilav**
The SCCA task can fail the builds if it detects Dockerfiles in our repo that reference external images. We're working on a solution with the SCCA team that will meet the needs of our open-source project. In the meantime, we're disabling this task on their recommendation.

I used the Build Images pipeline to confirm that the SCCA task passes when the `DisableDockerDetector` variable is set in the pipeline.

## Azure IoT Edge PR checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines and Best Practices
- [X] I have read the [contribution guidelines](https://github.com/azure/iotedge#contributing).
- [X] Title of the pull request is clear and informative.
- [X] Description of the pull request includes a concise summary of the enhancement or bug fix.

### Testing Guidelines
- [X] Pull request includes test coverage for the included changes.
- Description of the pull request includes 
	- [X] concise summary of tests added/modified
	- [X] local testing done.

### CODE REVIEW

File: builds/misc/templates/build-packages.yaml [hunk 1]
```diff
@@ -3,6 +3,9 @@ parameters:
     type: boolean
     default: false
 
+variables:
+  DisableDockerDetector: true
```
Comment (damonbarry) [inline builds/misc/templates/build-packages.yaml:7]: Adding this variable to a template doesn't really help (it's redundant), but also doesn't hurt.

**Review by damonbarry — APPROVED** [review]

File: builds/misc/templates/build-packages.yaml (hunk 1 above)
Comment (damonbarry) [reply builds/misc/templates/build-packages.yaml:7]: Actually, it looks like adding the variable to the template _is_ a problem:
image

**Review by damonbarry — APPROVED** [review]

**Review by and-rewsmith — CHANGES_REQUESTED** [review]
We are missing longhaul

**Review by and-rewsmith — APPROVED** [review]
Nevermind we have it

**Pull request merged**